tests/unittest_nodes.py

```diff
@@ -542,6 +542,19 @@ def test_str(self):
     def test_unicode(self):
         self._test("a")
 
+    @pytest.mark.skipif(
+        not PY38, reason="needs to be able to extract kind from ast.Constant"
+    )
+    def test_str_kind(self):
+        node = builder.extract_node(
+            """
+            const = u"foo"
+        """
+        )
+        self.assertIsInstance(node.value, nodes.Const)
```

Comment:
Can you use `assert isinstance(...)`? We try to move to pytest (at least for the newly added tests). The same for assertEqual below.

Reply:
sure!